### PR TITLE
Fixed broker CI build.

### PR DESCRIPTION
**Pull request by vadim-kovalyov**

### CODE REVIEW

File: edge-hub/test/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter.Test/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter.Test.csproj [hunk 1]
```diff
@@ -1,10 +1,6 @@
...
 
-  <PropertyGroup>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
-
-    <IsPackable>false</IsPackable>
-  </PropertyGroup>
+  <Import Project="..\..\..\netcoreappVersion.props" />
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
```
Comment (philipktlin) [inline edge-hub/test/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter.Test/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter.Test.csproj:6]: please also update to use 16.4.0
```suggestion
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.4.0" />
```

Comment (philipktlin) [reply edge-hub/test/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter.Test/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter.Test.csproj:6]: And please update xunit.runner.visualstudio to 2.4.1.  Also please update other projects to use 1 single version of moq (4.13.1) in this PR or another PR.

**Comment by philipktlin** [comment]
Another thing I found that there are projects referencing Microsoft.AspNetCore, Microsoft.AspNetCore, and Microsoft.NetCore.App. And some project file is not updated as what we have in master branch, e.g. TestAnalyzer project file still contains below property group.  Please verify all project files.

  <PropertyGroup Condition="'$(DotNet_Runtime)' != 'netcoreapp3.0'">
    <TargetFramework>netcoreapp2.1</TargetFramework>
  </PropertyGroup>

  <PropertyGroup Condition="'$(OS)|$(DotNet_Runtime)' == 'Unix|netcoreapp3.0'">
    <TargetFramework>netcoreapp3.0</TargetFramework>
  </PropertyGroup>

File: edge-hub/test/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter.Test/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter.Test.csproj (hunk 1 above)
Comment (vadim-kovalyov) [reply edge-hub/test/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter.Test/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter.Test.csproj]: I brought it back to a single 4.10 version. I wouldn't want to make solution wide changes in this PR.

**Pull request merged**